package.json

```diff
@@ -27,6 +27,7 @@
     "@navikt/nytt-i-rekrutteringsbistand": "^1.2.7",
     "@reach/menu-button": "^0.11.2",
     "@sentry/react": "^5.23.0",
+    "@testing-library/react": "^11.0.4",
```

Comment:
Kan muligens fjerne denne siden vi ennå ikke har noen komponenttester? Så kan den som eventuelt lager den første ta en research-runde for å finne hva som er best å bruke

Reply:
Fjerner